packages/mobile/App/ui/navigation/stacks/VaccineTableTabs.tsx

```diff
@@ -1,5 +1,5 @@
 import React, { ReactElement, useEffect } from 'react';
-import { ScreenOrientation } from 'expo-screen-orientation';
```

Comment:
Using `expo-screen-orientation` somehow crashes the app. Switched it to `react-native-orientation-locker` preserve the behaviour and not crashing it